### PR TITLE
initial skeleton for spark plugin dir, added icons to img dir

### PR DESCRIPTION
**Pull request by srlu94**

### CODE REVIEW

**Comment by molner** [comment]
LGTM

File: collectd-spark/README.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+* `spark_process`, either master or worker to differentiate master- and worker- specific metrics like master.apps and worker.coresFree 
+
+Metrics at the application level (endpoint /api/v1/applications) will contain the following dimension by default:
+
+* `cluster`, set to value corresponding to key "Cluster" in configuration file 
+
+Additional details:
+
+* `plugin` is always set to `Apache Spark`
```
Comment (charliesignalfx) [inline collectd-spark/README.md]: @srlu94 This has changed, please update accordingly.

File: collectd-spark/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{ "display_name":"Apache Spark",
+  "description": "Monitor Spark cluster and applications using the SignalFx collectd agent.",
+  "project_url": "https://github.com/signalfx/integrations/tree/master/collectd-spark",
+  "code": "https://github.com/signalfx/collectd-spark",
+  "featured": false,
+  "logo_large": "/images/repos/collectd-spark/img/integrations_apachespark.png",
+  "logo_small": "/images/repos/collectd-spark/img/integrations_apachespark%402x.png",
+  "data_signature": "plugin:\"Apache Spark\"",
```
Comment (charliesignalfx) [inline collectd-spark/meta.yaml]: I think this may need to change @srlu94 but please verify with @mbskinner

**Comment by charliesignalfx** [comment]
Please address my two comments about the plugin name.  Otherwise LGTM.  Let's get the dashboards imported, merge your code, and then we can merge this PR.

**Comment by charliesignalfx** [comment]
LGTM

**Pull request merged**